README.md

```diff
@@ -133,3 +133,13 @@ or rebuild everything if you are desperate (will take longer, but might solve ot
 ```bash
 docker-compose build
 ```
+
+#### Different networks:
+
+In order to start StableX for the Rinkeby network, overwrite certain environmental variables by adding common-rinkeby.env in the docker-compose, like this:
+
+```
+env_file:
+    - common.env
```

Comment:
I was hoping we could find a way where a different command could invoke the rinkeby version of our services. Something like:

`docker-compose stablex --env_file common-rinkeby.env`

Since that doesn't seem to be possble maybe it'd be better to use a separate target (e.g. stablex rinkeby)? We can probably reuse the existing code by using << yaml inclusion similar to what we do for `&rust-service`?